TableCell/index.jsx

```diff
@@ -16,11 +16,7 @@ const TableCellStateless = ({
     hovered,
   });
   return (
-    <div
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-      style={style}
-    >
+    <div onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave} style={style}>
```

Comment:
This is about as long as you'd want on a single line I think. Might be better to keep it the way it was because it makes it easier to refactor later (things like add/remove a prop).